src/glossary.rst

```diff
@@ -412,6 +412,29 @@ Glossary
       * :term:`job`
       * :term:`qualifier`
 
+   implicit task
+   naked task
```

Comment:
```suggestion
```

Reply:
What about changing the first line to "Implicit tasks (previously known as naked tasks)..."

Reply:
Could do, however, the terminology before was a mixture of, `dummy` (incorrect), `naked` (strange) and `naked-dummy` (really strange and somewhat suggestive).